app/aws/s3.py

```diff
@@ -60,8 +60,10 @@ def get_job_metadata_from_s3(service_id, job_id):
     return obj.get()["Metadata"]
 
 
-def remove_job_from_s3(service_id, job_id):
-    return remove_s3_object(*get_job_location(service_id, job_id))
+def remove_job_batch_from_s3(jobs):
```

Comment:
```suggestion
def remove_jobs_from_s3(jobs):
```
I just pluralized it.. and then remove batch but it is okay if we keep it.. just want to pluralize a function that performs a mass deletion. 😉